CHANGELOG.md

```diff
@@ -6,7 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [10.1.0] - 2021-06-7
+## [10.2.0]
+
```

Comment:
This newline shouldn't be here. And I think there are newlines missing later as well. See `yarn auto-changelog --validate` to validate this.